cmd/src/debug_comp.go

```diff
@@ -0,0 +1,228 @@
+package main
```

Comment:
This file might be better named `debug_compose.go` to better match the command name.